src/native/SystemApp.js

```diff
@@ -21,6 +22,12 @@ export function changeColorTheme(theme: string): Promise<void> {
 	return nativeApp.invokeNative(new Request('changeTheme', [theme]))
 }
 
+export function changeSystemLanguage(language: {code: string, languageTag: string}): Promise<void> {
+	return isDesktop()
```

Comment:
Not sure if we need to stub it here if we have implementation for apps but we can leave it like this for now.